src/app/core/project-service/project-service.service.ts

```diff
@@ -1,8 +1,74 @@
 import { Injectable } from '@angular/core';
+import { Headers, Http, Response } from '@angular/http';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/catch';
+import 'rxjs/add/observable/throw';
+import 'rxjs/add/operator/do';
 
 @Injectable()
 export class ProjectServiceService {
+    
+  _http;
 
-  constructor() { }
+  constructor(private http: Http) {
+    this._http = http;
+  }
+
+private handlError(error: Response){
+    let message = `Error status ${error.status} at ${error.url}`;
+    return Observable.throw(message);
+}
+
+getProjects(): Observable<any> {
```

Comment:
It would be nice to creta type (interface) for Project instead of `any`. Same for all methods here. I can give you some advices and samples for it if you need